interfaces/Portal/src/app/auth/permission.enum.ts

```diff
@@ -1,4 +1,4 @@
-import { PermissionEnum } from '../../../../../services/121-service/src/user/enum/permission.enum';
+import { PermissionEnum } from '@121-service/src/user/enum/permission.enum';
```

Comment:
We could get rid of the "Portal-local"-file now, I think.

It was only here to prevent having all these 'deep' imports everywhere in the Portal-code.

With these aliases, it serves no purpose anymore.

But I still don't like the naming of (a lot of) these 121-service `enum`s.
- Some are (inconsistently) post-fixed with `*Enum`
- Some are in plural form
- etc..

Reply:
As mentioned in the other comment, I only made auto-fix lint changes to the files. I'd rather avoid making other changes on this PR to avoid scope creep and rabbit holes.